test/onnx/torchlib/ops_test_common.py

```diff
@@ -593,6 +601,7 @@ def _capture_graph_and_evaluate_torch_script_evaluator(
         _ir_passes.add_torchlib_common_imports(onnx_model)
         _ir_passes.add_opset_imports(onnx_model)
         # Make sure the model is valid
+        version_converter.convert_version(onnx_model, target_version=opset_version)
```

Comment:
```suggestion
```

Reply:
This was needed to upgrade torchlib.common.ops to correct opset version, but doing this now id add_torchlib_common_imports